### DISCUSSION TITLE
computational complexity

### DISCUSSION THREAD

**Discussion by squarefaceyao** [lululxvi/deepxde#1103] [Q&A]
May I ask how to calculate the computational complexity of the minimum and average time I use to solve the problem with PINN?

**Comment by lululxvi** [comment]
Just run the code, and record the time?

**Reply by ShaikhaTheGreen** [reply]
I'm interested in this discussion as well!
Is there an equation to calculate the complexity of the code like in Eq. 29 in Section A.1 of this paper https://arxiv.org/pdf/2107.07871.pdf ?

How many order of magnitude is the difference in speed between DeepXDE (prediction) over FDM (Execution)? 

Looking forward for hearing your thoughts about this.

**Reply by lululxvi** [reply]
If you want to use flops for the complexity, then you can simply compute it manually for both DeepXDE and FDM.
